apps/sms/test/marionette/composer_test.js

```diff
@@ -37,6 +37,7 @@ marionette('Messages Composer', function() {
 
   var MOCKS = [
     '/mocks/mock_test_storages.js',
+    '/mocks/mock_test_blobs.js',
```

Comment:
I started to use image generation code in contacts mock as well - so moved it to a separate module.